docs/security/lcmaps-voms-authentication.md

```diff
@@ -123,20 +119,20 @@ The program edg-mkgridmap (found in the package `edg-mkgridmap`), used for authe
     | HTCondor-CE                 | `/etc/sysconfig/condor-ce`             | `condor-ce`                     |
     | GridFTP server              | `/etc/sysconfig/globus-gridftp-server` | `globus-gridftp-server`         |
 
-3.  If you are converting an HTCondor-CE host, remove the HTCondor-CE `GRIDMAP` configuration. Otherwise, skip to the next step.
+3.  If you are converting an HTCondor-CE host, remove the HTCondor-CE `GRIDMAP` configuration (`osg-configure` may have already done this for you). Otherwise, skip to the next step.
```

Comment:
It seems a little awkward to have the parenthetical here. Maybe a note before the inner list? Or just let line 129 (now) do its work?